bundle/jsky.app.ot/src/main/java/jsky/catalog/gui/CatalogNavigator.java

```diff
@@ -1,1243 +0,0 @@
-    private AbstractAction _saveWithImageAction = new AbstractAction(_I18N.getString("saveCatalogWithImage")) {
-        public void actionPerformed(ActionEvent evt) {
-            saveWithImage();
-        }
-    };
-
-    // Action to use for the "Save as HTML..." menu and toolbar items
-    private AbstractAction _saveAsHTMLAction = new AbstractAction(_I18N.getString("saveAsHTML")) {
-        public void actionPerformed(ActionEvent evt) {
-            saveAsHTML();
-        }
-    };
-
-    // Action to use for the "Print..." menu and toolbar items
-    private AbstractAction _printAction = new AbstractAction(_I18N.getString("print") + "...") {
-        public void actionPerformed(ActionEvent evt) {
-            print();
-        }
-    };
-
-    // Action to use for the "Back" menu and toolbar items
-    private AbstractAction _backAction = new AbstractAction(_I18N.getString("back")) {
-        public void actionPerformed(ActionEvent evt) {
-            back();
-        }
-    };
-
-    // Action to use for the "Forward" menu and toolbar items
-    private AbstractAction _forwAction = new AbstractAction(_I18N.getString("forward")) {
-        public void actionPerformed(ActionEvent evt) {
-            forward();
-        }
-    };
-
-    // Action to use for the "Add Row" menu item
-    private AbstractAction _addRowAction = new AbstractAction(_I18N.getString("addRow")) {
-        public void actionPerformed(ActionEvent evt) {
-            addRow();
-        }
-    };
-
-    // Action to use for the "Delete Rows..." menu item
-    private AbstractAction _deleteSelectedRowsAction = new AbstractAction(_I18N.getString("deleteSelectedRows")) {
-        public void actionPerformed(ActionEvent evt) {
-            deleteSelectedRows();
-        }
-    };
-
-    // Action to use for the "Query => Store => New Query..." menu item
-    private AbstractAction _storeNewQueryAction = new AbstractAction(_I18N.getString("new")) {
-        public void actionPerformed(ActionEvent evt) {
-            storeNewQuery();
-        }
-    };
-
-    // Action to use for the "Query => Delete => All" menu item
-    private AbstractAction _deleteAllQueryAction = new AbstractAction(_I18N.getString("all")) {
-        public void actionPerformed(ActionEvent evt) {
-            clearQueryList();
-        }
-    };
-
-
-    /**
-     * Return the top level catalog directory to use based on the value of the
-     * <em>jsky.catalog.directory</em> system property, which may be set to the name of
-     * the class implementing the CatalogDirectory interface. The default is
-     * to use the {@link jsky.catalog.astrocat.AstroCatConfig} class (The
-     * {@link jsky.catalog.skycat.SkycatConfigFile} class is another alternative).
-     * The static method in the given class named <em>getDirectory()</em> is called to
-     * return a reference to the top level CatalogDirectory.
-     */
-    public static CatalogDirectory getCatalogDirectory() {
```

Comment:
This method was guilty of many reflection issues in the past. It is not in use anymore